docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -278,27 +239,17 @@ When set, the input field offers visual indications that a feel expression is ex
 
 ###### Supported types
 
-Camunda Platform 8 supports `feel` on the following input types:
+The property `feel` is supported on the following input types:
 
 - `String`
 - `Text`
 
 #### Bindings
```

Comment:
I can't find the binding type `bpmn:Message#property` documented anywhere. Are we aware of that?

Reply:
Hmm I believe I overlooked this when I implemented the feature 🤡